src/routes.ts

```diff
@@ -386,10 +386,12 @@ async function setupPage(ctx: RequestContext, params: BaseRequestAPICall, browse
 const browserRequest = async (ctx: RequestContext, params: BaseRequestAPICall) => {
   const oneTimeSession = params.session === undefined
   const sessionId = params.session || UUIDv1()
+  const proxy = params.proxy;
```

Comment:
Off the top of my head, I feel like this is more complicated than this... Have you tested this thoroughly?

Reply:
The logic behind it was added already. I just made it pass the proxy to the session